formbricks/formbricks
# feat: support for make and n8n integration states

## What does this PR do?
Adds Integration States visibility for `make` and `n8n`:
![image](https://github.com/formbricks/formbricks/assets/55556994/6d54c7ce-165d-4ab8-a145-6c30c44a71ed)

We now need to pass this as a request body parameter in the make and n8n apps as well!

## Type of change

## How should this be tested?
- Use the create webhook API endpoint and pass in a "source" key in the json body with value as any of the allowed sources: "make", "n8n", "zapier", "user".
- Now once this webhook is created for the given source, it should reflect the same in the connection state.

## Checklist


### Required


### Appreciated
